client/src/components/organization/index.js

```diff
@@ -25,8 +25,8 @@ const sampleImg = require('./sample.jpg')
 
 class Organization extends Component {
   componentWillMount() {
-    const { fetchOrganization } = this.props
-    fetchOrganization(13)
+    const { fetchOrganization, match } = this.props
```

Comment:
is this to grab the id of the organization that is logged in? If so, is there a way to just map the organizations id to the props in the container of this component?

Reply:
This change makes the page able to grab the ID from the url so that we can fetch the requested organization by ID. Before, we weren't checking for the requested ID and were just using '13' as a placeholder.